scripts/travis/test.sh

```diff
@@ -1,11 +1,18 @@
 #!/usr/bin/env bash
 
 set -e
+set -x
 
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 OS=$("${SCRIPTPATH}/../ostype.sh")
 ARCH=$("${SCRIPTPATH}/../archtype.sh")
 
+if [[ "${ARCH}" != "arm" ]]; then
+    curl -sL -o ~/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
+    chmod +x ~/gimme
+    eval $(~/gimme $("${SCRIPTPATH}/../get_golang_version.sh"))
+fi
+
```

Comment:
The `travis/test.sh` is called only after `travis/build.sh` which already have this installed.

Reply:
Required due to environment scope of gimme.  Gimme reuses go install if it exists already.